README.md

```diff
@@ -316,7 +316,8 @@ Management can work with Legal to determine Intellectual Property concerns.
 
 Many package managers include a feature to make the applied license machine readable. Use these! An example for [Maven](https://maven.apache.org/pom.html#Licenses):
 
- ```<licenses>
+```xml
```

Comment:
the opening tag was omitted before, as it was interpreted as the language of the code